parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ColumnChunkPageWriteStore.java

```diff
@@ -316,7 +346,14 @@ private int toIntWithCheck(long size) {
       return (int)size;
     }
 
-    private void mergeColumnStatistics(Statistics<?> statistics) {
+    private void mergeColumnStatistics(Statistics<?> statistics, SizeStatistics sizeStatistics) {
+      Preconditions.checkState(totalSizeStatistics != null, "Aggregate size statistics should not be null");
```

Comment:
If the `totalSizeStatistics` is set as final. The checking here is not needed anymore.